### PR TITLE
Fix grammatical issues in accepted proposals

### PR DESCRIPTION
**Pull request by colinodell**
Some of the issues are minor (excessive or missing commas) while others are slightly more glaring (incorrect tense, wrong preposition, missing/extra words, singular vs. plural, etc.)

### CODE REVIEW

File: accepted/PSR-1-basic-coding-standard.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ if (! function_exists('bar')) {
 
 Namespaces and classes MUST follow an "autoloading" PSR: [[PSR-0], [PSR-4]].
 
-This means each class is in a file by itself, and is in a namespace of at
+This means each class is in a file by itself and is in a namespace of at
```
Comment (Crell) [inline accepted/PSR-1-basic-coding-standard.md]: I wouldn't call this one incorrect as much as a stylistic difference.

File: accepted/PSR-3-logger-interface.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ Users of loggers are referred to as `user`.
 
 ### 1.3 Context
 
-- Every method accepts an array as context data. This is meant to hold any
+- Every method accepts an array of context data. This is meant to hold any
```
Comment (Crell) [inline accepted/PSR-3-logger-interface.md]: Debatable. I think it works either way, so stet.

File: accepted/PSR-6-cache.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ persisted immediately by the pool. A Pool object MAY delay persisting a deferred
 cache item in order to take advantage of bulk-set operations supported by some
 storage engines. A Pool MUST ensure that any deferred cache items are eventually
 persisted and data is not lost, and MAY persist them before a Calling Library
-requests that they be persisted. When a Calling Library invokes the commit()
+requests that they are persisted. When a Calling Library invokes the commit()
```
Comment (Crell) [inline accepted/PSR-6-cache.md]: Stylistic difference, I think both are valid here. Stet.

**Comment by Crell** [comment]
Other than those noted above this seems fine to me.  Thanks!

**Comment by colinodell** [comment]
Thanks for the feedback @Crell!  I've reverted those three changes, squashed, and force-pushed.

**Comment by pmjones** [comment]
For my part, the edits to PSR-4 are acceptable.

@Seldaek @weierophinney may wish to review the edits to their work as well.

**Comment by Seldaek** [comment]
Looks good to me, thanks for the ping.

**Comment by michaelcullum** [comment]
I'll merge this once I've had some form of a-okay from one of the PSR-7 team.

**Comment by weierophinney** [comment]
:+1: from me!

**Comment by michaelcullum** [comment]
Sorry for the delay, I missed Matthew's reply

**Pull request merged**